### DISCUSSION TITLE
Product roadmap

### DISCUSSION THREAD

**Discussion by rwxzig** [OrchardCMS/OrchardCore#14524] [General]
1. Is there any roadmap on features that are discussed to be implemented, probably with different priority level and future releases?

2. Is there any donating account/link for users who want to donate to the project?

**Comment by hishamco** [comment]
I remember this is the roadmap https://github.com/OrchardCMS/OrchardCore/wiki/Roadmap but it's not been updated for a while, but I think it's good to plan the features in advance
